draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -781,6 +781,22 @@ Verifiers need to ensure that they are not using expired or revoked keys
 for signature verification using reasonable and appropriate means for the given
 key-distribution method.
 
+## Selective Disclosure without Holder Binding
+
+The Issuers and the Verifiers should be aware that when Holder Binding is not present,
+any entity that is not the original receiver can be presenting the Combined Format for Presentation.
+When doing so, that entity may be applying selective disclosure to the SD-JWT
+that has already been previously selectively disclosed.
```

Comment:
```suggestion
When doing so, that entity may remove Disclosures such that the receiver
learns only a subset of the claims contained in the original Combined Format for Presentation.
```

Reply:
This seems better, maybe position "receiver" in terms of a "next verifier" to emphasis the "enforcement" comment you made earlier.